src/blind.rs

```diff
@@ -15,12 +15,12 @@
 //! # Transactions Blinding
 //!
 
-use std::{self, fmt};
+use std::{self, fmt, collections::BTreeMap};
```

Comment:
In 8b41c1327d8af43f2196ce97d230c5a0719d74c3:

I think you should use `HashMap` everywhere instead of `BTreeMap`. It doesn't really make a difference at all but you never iterate over this map directly (you always iterate over the Vec of tx inputs) and `HashMap` is a little bit easier to use and potentially more efficient.